screwdriver.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# Job definition block
+# "main" is a default job that all pipelines have
+jobs:
+  # Jobs are defined by name.
+  # All pipelines have "main" implicitly defined. The definitions in your screwdriver.yaml file will override the
+  # implied defaults.
+  main:
+    # Docker image to use as the desired build container. This typically takes the form of repo_name:tag_label
+    # 
+    # Of the many buildpack-deps images, this example uses the xenial tag
+    # (Source: https://hub.docker.com/r/library/buildpack-deps/)
+    image: buildpack-deps:xenial
```

Comment:
I guess not. I wasn't aware that it pulled from the `latest` tag if one isn't specified. 

I'll amend the comment